### PR TITLE
Fix showing current result/state filters on overview page

### PR DESCRIPTION
**Pull request by Martchus**
There's no ticket, just something I noticed when working on https://progress.opensuse.org/issues/154261.

### CODE REVIEW

File: t/ui/10-tests_overview.t [hunk 1]
```diff
@@ -497,8 +498,8 @@ subtest 'filtering by job group' => sub {
             name => 'SLE 15 SP5 development'
         });
 
-    my $get_text = sub {
-        $driver->get(shift);
+    my $get_text = sub ($url) {
```
Comment (kalikiana) [inline t/ui/10-tests_overview.t:501]: Isn't this just a style change? I don't see anything here that changes behavior

Comment (Martchus) [reply t/ui/10-tests_overview.t:501]: Yes, this is just a style change. I first thought I could reuse this function so I changed it to our current style as I had to move it around. As it turned out that I couldn't reuse it I still kept the style change (as separate commit).

Comment (kalikiana) [reply t/ui/10-tests_overview.t:501]: I suggest renaming the PR then

Comment (kalikiana) [reply t/ui/10-tests_overview.t:501]: Hrm. Looking at the commits `return filter.replace(/_/g, ' ');` is a functional fix, though?

Comment (kalikiana) [reply t/ui/10-tests_overview.t:501]: Okay, so I guess the part I was missing is this the `return` is required if you also use curly parens

**Review by kalikiana — APPROVED** [review]

File: t/ui/10-tests_overview.t (hunk 1 above)
Comment (Martchus) [reply t/ui/10-tests_overview.t:501]: Yes. The functional and the style changes are in separate commits so this should be clear.

**Review by okurz — APPROVED** [review]

File: t/ui/10-tests_overview.t (hunk 1 above)
Comment (kalikiana) [reply t/ui/10-tests_overview.t:501]: Well, it wasn't 😅 I mentioned it because it stopped me approving, and sometimes you ask about unreviewed PR's

**Pull request merged**